src/components/map/Map.tsx

```diff
@@ -135,7 +135,9 @@ class LeafletMap extends React.Component<IMapProps, IMapState> {
                     <NodeLayer
                         nodes={visibleNodes}
                     />
-                    <PopupLayer />
+                    <PopupLayer
+                        map={this.map}
+                    />
```

Comment:
Would it be nicer to just pass a setView function in the props? Feels a bit unnecessary to pass the whole map store.